frontend/src/components/GlobalProviders/RouteProvider/RouteProvider.jsx

```diff
@@ -46,6 +47,17 @@ const RouteProvider = () => {
               )
             }
           />
+
+          <Route
+            path="/recommend/restaurants/:location"
+            element={
+              isAuthenticated ? (
+                <RestaurantRecommendations/>
```

Comment:
If you leave out the `<AuthenticatedProvider />` you will lose the header component which is used to allow user to logout, change theme, go to profile page etc.

See the global providers as examples how to have children components in React:
![image](https://github.com/UOA-CS732-SE750-Students-2024/project-group-careless-crayfish/assets/29388401/fead2bd3-9ffe-49e3-b0a3-e44ae73e7d5b)
![image](https://github.com/UOA-CS732-SE750-Students-2024/project-group-careless-crayfish/assets/29388401/e4e6448a-4d68-49b3-98f9-2a899529d89e)